test/go-tests/testsuite_gke_test.go

```diff
@@ -0,0 +1,45 @@
+package go_tests
+
+import (
+	"testing"
+)
+
+func Test_GKE(t *testing.T) {
+	// Common Tests
```

Comment:
would make sense to create `testsuite_common_test.go` which is run for each platform?

Reply:
I am not sure if this works but I can try. It's a bit tricky with those test suites since they are actually the go test functions but if I add a grouping function below I don't know how the test results will behave.

Reply:
We can iterate in a follow up if we see the need for such a thing :)